introduction.md

```diff
@@ -0,0 +1 @@
+Hello fellow ninjas! My name is Sander and I'm here to sharpen my blade!
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.